vscode-client/package.json

```diff
@@ -346,6 +549,34 @@
           "type": "boolean",
           "default": false
         },
+        "cquery.highlighting.bold.namespaces": {
+          "type": "boolean",
```

Comment:
Feel free to set some of these to true by default where you feel is appropriate, I don't expect users to customize this on their own.

Reply:
I wonder if is possible that, instead of having `"cquery.highlighting.bold.namespaces"` for each symbol kind, use a "cquery.highlighting.bold" with `"type": "object"` ( JS object, i don't know its TS counterpart)

Reply:
Such as:
```typescript
"cquery.highlighting.enabled": {
"Namespace": true,
"Class": false,
}
```

I prefer singular form and don't change the case as used in clang symbol kind or LSP symbol kind

Reply:
FWIW the advantage of the current approach is that all customization for a highlight kind is colocated, ie, make namespaces bold/italic with color xyz can be on sequential lines.